lib/ASRouter.jsm

```diff
@@ -1880,6 +1880,21 @@ class _ASRouter {
       case ra.OPEN_APPLICATIONS_MENU:
         UITour.showMenu(target.browser.ownerGlobal, action.data.args);
         break;
+      case ra.HIGHLIGHT_FEATURE:
+        const highlight = await UITour.getTarget(
+          target.browser.ownerGlobal,
+          action.data.args
+        );
+        if (highlight) {
+          await UITour.showHighlight(
+            target.browser.ownerGlobal,
+            highlight,
+            // aEffect
```

Comment:
Forgot to remove this or intentional?

Reply:
Intentional. I thought „none“ as a string was kind of random, it’s just the argument name as a comment.